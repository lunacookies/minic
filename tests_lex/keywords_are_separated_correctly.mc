func
func_
123func
_func
