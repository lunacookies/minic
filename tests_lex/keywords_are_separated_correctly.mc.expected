{
	FUNC 0..4
	IDENTIFIER 5..10
	NUMBER 11..14
	FUNC 14..18
	IDENTIFIER 19..24
}
